Package.swift

```diff
@@ -80,7 +80,7 @@ let package = Package(
                         "Nimble",
                         .product(name: "SnapshotTesting", package: "swift-snapshot-testing")
                     ],
-                    exclude: ["Templates/__Snapshots__"],
+                    exclude: ["Templates/__Snapshots__", "Data/__Snapshots__", "TestPlans"],
```

Comment:
I ran `swift build` and got some warnings, so I realized these were missing from here.